start-pantheon-tweaks.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/bash
+# Wrapper script to get/set GSettings from the host in the sandboxed Pantheon Tweaks.
+# Originally from https://github.com/flathub/ca.desrt.dconf-editor/blob/master/start-dconf-editor.sh
+
+trap "finalize; exit 1" SIGINT
+
+finalize()
+{
+  for dir in "${HOST_XDG_DATA_DIRS//:/ }"; do
+    unlink $dir/glib-2.0/schemas
+    rmdir $dir/glib-2.0
+    rmdir $dir
```

Comment:
Why delete these?

Reply:
Because these are the temporary symlink and directories that this script created.

```bash
  9   for dir in "${HOST_XDG_DATA_DIRS//:/ }"; do
 10     unlink $dir/glib-2.0/schemas # unlink the temporary symlink created in the line 36
 11     rmdir $dir/glib-2.0 # delete the temporary directory created in the line 35
 12     rmdir $dir # delete the temporary directory created in the line 35
 13   done
 14
 15   rmdir "$bridge_dir" # remove the temporary directory created in the line 23
```